azkaban-db/src/main/java/azkaban/db/AzDBUtil.java

```diff
@@ -18,5 +18,6 @@
 
 public class AzDBUtil {
 
-  static final int MAX_DB_RETRY_COUNT = 5;
+  // A very big Integer
+  static final int MAX_DB_RETRY_COUNT = 999999;
```

Comment:
if it keeps reconnecting for too long, how should we get alerted?

Reply:
This is a very good concern. I haven't thought about this much. The simple and straightforward solution to report retry Attempt count. If this number is too high, we should get alerted.

Reply:
Good discussion.
Yes, some alerts would be useful.
+ @li-afaris